runtimes/pythonrt/py.go

```diff
@@ -36,9 +36,31 @@ func createTar(fs fs.FS) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
+type Version struct {
+	Major int
+	Minor int
+}
+
 type exeInfo struct {
-	Exe     string
-	Version string
+	Exe           string
+	VersionString string
+	Version       Version
+}
+
+func parsePyVersion(s string) (major, minor int, err error) {
+	// Python 3.12.2
+	const prefix = "Python "
+	if len(s) < len(prefix) {
```

Comment:
I think there is no additional value in check length.
Maybe just move Python as %s, e.g. "%s %d.%d" to scanf?

Reply:
Changed to `strings.HasPrefix`